pkg/transformers/config/transformerconfig.go

```diff
@@ -56,6 +56,7 @@ func MakeDefaultConfig() *TransformerConfig {
 // sortFields provides determinism in logging, tests, etc.
 func (t *TransformerConfig) sortFields() {
 	sort.Sort(t.NamePrefix)
+	sort.Sort(t.NameSuffix)
```

Comment:
This is irrelevant. FieldSpecs related to suffix (t.NameSuffix) should have been deprecated long time ago. The transformer only uses [one set](https://github.com/kubernetes-sigs/kustomize/blob/master/plugin/builtin/prefixsuffixtransformer/PrefixSuffixTransformer.go#L21).

So when the transformer is initialized, it only passes one t.NamePrefix: [here](https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/target/kusttarget_configplugin.go#L251)

Reply:
Sorry, I wasn't aware of the semantics, it was mostly just convenient to use from the existing test and I initially got unexpected results without it.